docs/dockerhub.md

```diff
@@ -0,0 +1,106 @@
+# jpo-ode
+
+## GitHub Repository Link
+https://github.com/usdot-jpo-ode/jpo-ode
+
+## Purpose
+The purpose of the Operational Data Environment is to ingest, process and disseminate connected vehicle data in the form of messages defined in J2735. The primary program utilizes a number of internal modules and external submodules.
+
+## How to pull the latest image
+The latest image can be pulled using the following command:
+> docker pull usdotjpoode/jpo-ode:latest
+
+## Required environment variables
+The image expects the following environment variables to be set:
+- DOCKER_HOST_IP
+- DOCKER_SHARED_VOLUME
+
+## Direct Dependencies
+The ODE will fail to start up if the following containers are not already present:
+- Kafka
+- Zookeeper (relied on by Kafka)
+
+## Indirect Dependencies
+Some functionality will be unreachable without the participation of the following programs (except by directly pushing to kafka topics):
+- asn1_codec
+- jpo-cvdp
+- jpo-security-svcs
+- jpo-sdw-depositor
+- jpo-s3-deposit
+
+## Example docker-compose.yml with direct dependencies:
```

Comment:
Let's swap this header to a "Configuration" and add in a note in here about "for further configuration options see GitHub repo with the link to it on "GitHub". Then we can subheader the example doc below